tests/integration/integration_steps/cdf_steps.py

```diff
@@ -56,64 +59,150 @@ def push_data_to_cdf(time_series_data: list[TimeSeries], cognite_client: Cognite
     sleep(5)
     return time_series_data_points_pushed
 
+
 def create_subscription_in_cdf(time_series_data: list[TimeSeries], sub_name: str, cognite_client: CogniteClient):
     ts_external_ids = [ts.external_id for ts in time_series_data]
-    sub = DataPointSubscriptionWrite(sub_name, partition_count=1, time_series_ids=ts_external_ids, name="Test subscription")
+    sub = DataPointSubscriptionWrite(
+        sub_name, partition_count=1, time_series_ids=ts_external_ids, name="Test subscription"
+    )
     return cognite_client.time_series.subscriptions.create(sub)
 
+
 def create_data_model_in_cdf():
     # Create a data model in CDF
     pass
 
+
 def update_data_model_in_cdf():
     # Update a data model in CDF
     pass
 
 
 def compare_timestamps(timestamp1: datetime, timestamp2: datetime) -> bool:
-    return timestamp1.replace(microsecond=0) == timestamp2.replace(microsecond=0)
+    return timestamp1 == timestamp2
```

Comment:
Is there a reason we needed to get rid of the microseconds  this could have repercussion.